### ISSUE TITLE
Training:How to change the lr to make the weights be stronger?Who can give me a piece of advice?Thanks! 

### ISSUE THREAD

**Issue by gaoruihua111** [lightvector/KataGo#640]

**Comment by Ishinoshita** [comment]
Pieces of advice: 
- give details (what exactly are you doing, what is your current learning rate, setings, training step numbers,  etc. ...). A title is not enough. If you expect devs to take the pain to answer you, take the pain to explain them better. 
- again, discord #help a better place for such vague help demands ...

**Comment by gaoruihua111** [comment]
I am using the -lr 0.8 to train and the step numbers are 150k

**Comment by gaoruihua111** [comment]
@lightvector

**Comment by lightvector** [comment]
This questions is too still vague to be answerable, you have not provided enough information for anyone to understand what your goal is, what you've tried so far, what your results are, what training data, what size neural nets, etc etc, and what kind of problems you are running or why you think your results are not good into that motivates you to ask this question in the first place. 

Instead of opening so many separate small issues on Github, please consider what Ishinoshita has said. Join the Discord https://discord.gg/45EWcZu7 and find someone to chat with in real-time where you can have a quicker back and forth discussion. If part of the difficulty is that English is not your most fluent language, then although this Discord is not the most active, you may sometimes be able to find people to help you in other languages there too.
